utils/README.md

```diff
@@ -0,0 +1,30 @@
+# Miscellaneous utilities for iotlab-cli-tools
+
+## Bash-completion
+
+The script `iotlabcli-bash-completion.sh` is available to complete the
+`iotlab-*` commands.  It is compatible with the standard bash-completion
+mechanism, as many commands are in [https://github.com/scop/bash-completion].
+The scripts only needs to be sourced at runtime from command-line:
+
+    source iotlabcli-bash-completion.sh
+
+Then, bash is able to autocomplete the iotlab-* commands:
+
+    iotlab-experiment submit --<press tab key here>
+
+
+One also wants to install it on a whole system, to be automatically available
+for all users:
+
+    sudo install -m644 iotlabcli-bash-completion.sh /usr/share/bash-completion/completions/iotlab-experiment
+    for command_name in iotlab-admin admin-cli  \
+                        iotlab-auth auth-cli  \
+                        experiment-cli  \
```

Comment:
This line should be removed.

Reply:
If you do, you won't support `*-cli` commands at all.  Your modifications in #30 show that you are precisely expecting that.  If that is the case, I agree with them.

Reply:
Yes, and this is expected! *-cli commands are now deprecated. Why do we want to have this cool new feature with them ? I hope it will enforce people to use the new names :)